src/dataset.rs

```diff
@@ -399,6 +399,30 @@ impl Dataset {
         }
         Ok(Transaction::new(self))
     }
+
+    pub fn overview_count(&self, band_index: isize) -> Result<i32> {
+        unsafe {
+            let c_band = gdal_sys::GDALGetRasterBand(self.c_dataset, band_index as c_int);
+            if c_band.is_null() {
+                return Err(_last_null_pointer_err("GDALGetRasterBand"));
+            }
+            Ok(gdal_sys::GDALGetOverviewCount(c_band))
+        }
+    }
+
+    pub fn overview_get(&self, band_index: isize, overview_index: isize) -> Result<RasterBand> {
```

Comment:
🤓 idiomatic rust omits the `get` from getters: https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter